snuba/subscriptions/data.py

```diff
@@ -111,10 +136,109 @@ def to_dict(self) -> Mapping[str, Any]:
         raise NotImplementedError
 
 
+@dataclass(frozen=True, kw_only=True)
+class RPCSubscriptionData(SubscriptionData):
+    """
+    Represents the state of an RPC subscription.
+    """
+
+    table_request: str
+
+    def build_request(
```

Comment:
returning a snuba `Request` object here does not make sense. The RPC implementations will build whatever requests are necessary and execute them